docs/standards.md

```diff
@@ -25,7 +25,7 @@ XDM incorporates or builds a number of existing standards and specifications.
 Prior to XDM, Adobe defined and standardized the Extensible Metadata Platform (XMP) throughout its Creative Cloud products.
 The vocabularies and operations of XMP entities are essential for those products as is XDM for the wider experience technology ecosystem.
 
-In XDM, these two efforts are integrated via the Resource Description Framework (RDF), which is a Word Wide Web Consortium (W3C) standard for data model interchange.
+In XDM, these two efforts are integrated via the Resource Description Framework (RDF), which is a World Wide Web Consortium (W3C) standard for data model interchange.
 XMP models are, at their core, RDF models expressed in an XML-based serialization.
```

Comment:
And the fact is that XMP (and RDF) is serialization agnostic.  The new  ISO 16684-3 provides for a JSON-LD-based serialization of the XMP data model.